ProcessLib/SmallDeformation/SmallDeformationProcessData.h

```diff
@@ -60,8 +63,25 @@ struct SmallDeformationProcessData
     //! Assignments are not needed.
     void operator=(SmallDeformationProcessData&&) = delete;
 
+    bool isElementDeactivated(const std::size_t element_id)
+    {
+        if (!material_ids)
+            return false;
+        if (deactivated_material_subdomains.empty())
+            return false;
+
+        return (std::find(deactivated_material_subdomains.begin(),
```

Comment:
maybe the vector could be ordered and a binary search could be used.

Reply:
Done.